### ISSUE TITLE
Playnite imports demos from Origin as a complete games

### ISSUE THREAD

**Issue by benini02** [JosefNemec/Playnite#718]
The different demos of Origin, there are appearing as a complete game in the Playnite library.

**Comment by Continous** [comment]
I'm not sure it'd be a good idea to exclude them either. Rather, it may be a better idea to add a default tag for Demos.

**Comment by benini02** [comment]
Yes, JosefNemec, I am referring game trials. And I agree with Continuous as long as there is way to differentiate if is a complete game or is a trial/demo.
